api/app/Http/Responder/ArticleResponder.php

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: redjik
+ * Date: 31.07.15
+ * Time: 11:57
+ */
+
+namespace App\Http\Responder;
+
+use App\Models\Article;
+
+class ArticleResponder extends Responder
+{
+    public function paginatedCollection($items, $offset = null, $totalCount = null, array $parameters = [])
+    {
+        /** @var Article[] $items */
+        foreach ($items as $item) {
+            $item->addHidden('content');
+        }
+
+        return parent::paginatedCollection($items, $offset, $totalCount, $parameters);
+    }
+}
```

Comment:
This is not an appropriate location to hide a field. The view logic is the concern of the transformer, not the responder. I want to avoid creating custom implementations for routes of the responder class.